### DISCUSSION TITLE
Error when importing twilio-chat - Module not found: Can't resolve 'babel-runtime/core-js/array/from'

### DISCUSSION THREAD

**Discussion by cdedreuille** [vercel/next.js#16231] [Help] [answered]
Hi,

I'm moving my app from create-react-app to Nextjs and I'm running into a weird bug when installing twilio-chat.
Every time I run the code I have this error:
```
error - ./node_modules/twilio-chat/browser/logger.js
Module not found: Can't resolve 'babel-runtime/core-js/array/from' in '/sandbox/node_modules/twilio-chat/browser'
```

I created a sandbox with a basic nextjs set up to reproduce the error.
https://codesandbox.io/s/condescending-solomon-25okm?file=/pages/index.js

I tried to install @babel/runtime and other packages from @babel but nothing works.
Any idea what could go wrong here?

**Comment by jamesmosier** [comment, marked as answer]
I think the issue is how the `twilio-chat` library defines what the package entry points are. Next is grabbing the `browser` entry but that is not working. `lib` seems to work though.

`twilio-chat/package.json`
```json
{
  "name": "twilio-chat",
  "version": "4.0.0",
  "description": "Twilio Chat service client library",
  "main": "lib/index.js",
  "browser": "browser/index.js",
  "types": "./lib/client.d.ts",
...
```

 If you change the import to `import { Client as ChatClient } from 'twilio-chat/lib';` it works.

**Reply by cdedreuille** [reply]
Amazing !! Thanks a lot for looking into this. It worked.
